src/config/constants/tokens.ts

```diff
@@ -1,1959 +1,1692 @@
-      56: '0xDB021b1B247fe2F1fa57e0A87C748Cc1E321F07F',
-      97: '',
-    },
-    decimals: 9,
-    projectLink: 'https://www.ampleforth.org/',
-  },
-  o3: {
-    symbol: 'O3',
-    address: {
-      56: '0xEe9801669C6138E84bD50dEB500827b776777d28',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://o3swap.com/',
-  },
-  hai: {
-    symbol: 'HAI',
-    address: {
-      56: '0xaA9E582e5751d703F85912903bacADdFed26484C',
-      97: '',
-    },
-    decimals: 8,
-    projectLink: 'https://hacken.io/',
-  },
-  htb: {
-    symbol: 'HTB',
-    address: {
-      56: '0x4e840AADD28DA189B9906674B4Afcb77C128d9ea',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://www.hotbit.io/',
-  },
-  woo: {
-    symbol: 'WOO',
-    address: {
-      56: '0x4691937a7508860F876c9c0a2a617E7d9E945D4B',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://woo.network/',
-  },
-  $dg: {
-    symbol: '$DG',
-    address: {
-      56: '0x9Fdc3ae5c814b79dcA2556564047C5e7e5449C19',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://decentral.games/',
-  },
-  safemoon: {
-    symbol: 'SAFEMOON',
-    address: {
-      56: '0x8076C74C5e3F5852037F31Ff0093Eeb8c8ADd8D3',
-      97: '',
-    },
-    decimals: 9,
-    projectLink: 'https://safemoon.net/',
-  },
-  axs: {
-    symbol: 'AXS',
-    address: {
-      56: '0x715D400F88C167884bbCc41C5FeA407ed4D2f8A0',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://axieinfinity.com/',
-  },
-  c98: {
-    symbol: 'c98',
-    address: {
-      56: '0xaec945e04baf28b135fa7c640f624f8d90f1c3a6',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://coin98.com/',
-  },
-  pots: {
-    symbol: 'POTS',
-    address: {
-      56: '0x3Fcca8648651E5b974DD6d3e50F61567779772A8',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://moonpot.com/',
-  },
-  gnt: {
-    symbol: 'GNT',
-    address: {
-      56: '0xf750a26eb0acf95556e8529e72ed530f3b60f348',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://www.greentrusttoken.com/',
-  },
-  rusd: {
-    symbol: 'rUSD',
-    address: {
-      56: '0x07663837218a003e66310a01596af4bf4e44623d',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://appv2.rampdefi.com/#/',
-  },
-  bp: {
-    symbol: 'BP',
-    address: {
-      56: '0xacb8f52dc63bb752a51186d1c55868adbffee9c1',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://www.bunnypark.com/',
-  },
-  sfund: {
-    symbol: 'SFUND',
-    address: {
-      56: '0x477bc8d23c634c154061869478bce96be6045d12',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://seedify.fund/ ',
-  },
-  naos: {
-    symbol: 'NAOS',
-    address: {
-      56: '0x758d08864fb6cce3062667225ca10b8f00496cc2',
-      97: '',
-    },
-    decimals: 18,
-    projectLink: 'https://naos.finance/',
-  },
+  return serializedTokens
 }
 
-export default tokens
+export default tokens()
```

Comment:
There are a few different ways to use the tokens in this file:

`import tokens from './tokens'`
Calls `tokens` function on ln 1678 which returns a list of `Token` entities, depending on the chain ID env variable

`import { mainnetTokens, testnetTokens } from './tokens'`
Directly returns `Token` entities from the mainnetTokens or testnetTokens lists, used when you want to access a specific network's token

`import { serializeTokens } from './tokens'`
Calls `tokens` func but returns serialized versions of all tokens.